packages/form-js-viewer/src/core/index.js

```diff
@@ -5,6 +5,8 @@ import FormFieldRegistry from './FormFieldRegistry';
 import importModule from '../import';
 import renderModule from '../render';
 
+export { FormFieldRegistry };
```

Comment:
If possible I'd like to avoid mixing default and named exports. What would you like to accomplish here?

Reply:
We're reusing the `FormFieldRegistry` in the editor, that's why it must be exported.

Reply:
Why not expose it, i.e. as a base module?

Reply:
Or is the registry a different thing in the viewer?

Reply:
The registry is the same in both viewer and editor. What do you mean by exposing it as a base module?